src/vmm.c

```diff
@@ -26,6 +26,10 @@ extern char _guest_initrd_image_end[];
 /* seL4CP will set this variable to the start of the guest RAM memory region. */
 uintptr_t guest_ram_vaddr;
 
+/* @jade: find a better number */
+#define MAX_IRQ_CH 32
```

Comment:
`MAX_IRQ_CH` should be 62 as per https://github.com/BreakawayConsulting/sel4cp/blob/main/docs/manual.md#protection_domain.

Reply:
This is actually already defined in `microkit.h` with `MICROKIT_MAX_CHANNELS`.